homeassistant/components/satel_integra.py

```diff
@@ -71,14 +74,16 @@
     """Set up the Satel Integra component."""
     conf = config.get(DOMAIN)
 
-    zones = conf.get(CONF_ZONES)
+    zones = conf.get(CONF_ZONES) if conf.get(CONF_ZONES) is not None else []
```

Comment:
The config schema says that zones should be a dict. Why should we default to empty list?

The default value should be moved to the config schema, is possible, eg if we can use empty dict insted.

Reply:
Makes sense, will change it in next iteration.